### PR TITLE
firstJencinsTestCreatePipeline

### PR DESCRIPTION
**Pull request by IrynaOkun**
creatingPipelineTest

### CODE REVIEW

File: src/test/java/OkunCreatePipeline.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+
+        String actualResult = getDriver().findElement(By.xpath("//h1"))
+                .getText();
+
+        deleteProjectPipeline();
+
+        Assert.assertEquals(actualResult, descriptionOfProject);
+
+
```
Comment (anonymoustuna) [inline src/test/java/OkunCreatePipeline.java:45]: Empty lines.

File: src/test/java/OkunCreatePipeline.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    }
+
```
Comment (anonymoustuna) [inline src/test/java/OkunCreatePipeline.java:47]: Empty line.

File: src/test/java/OkunCreatePipeline.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+                .sendKeys(pipelineProjectTitle);
+        getDriver().findElement(By.xpath("//div[contains(text(),'Orchestrates long-running')]"))
+                .click();
+        getDriver().findElement(By.id("ok-button"))
+                .click();
+        getDriver().findElement(By.xpath("//textarea"))
+                .sendKeys(descriptionOfProject);
+        getDriver().findElement(By.xpath("//div[@class='bottom-sticker-inner']"))
+                .click();
+
```
Comment (anonymoustuna) [inline src/test/java/OkunCreatePipeline.java:36]: Empty line.

File: src/test/java/OkunCreatePipeline.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+                .click();
+        getDriver().switchTo().alert().accept();
+
+    }
+
+    @Test
+    public void testPipelineCreate() {
+        final String pipelineProjectTitle = "Bedryks";
+        final String descriptionOfProject = "Pipeline Bedryks";
+
```
Comment (anonymoustuna) [inline src/test/java/OkunCreatePipeline.java:22]: Empty line.

File: src/test/java/OkunCreatePipeline.java [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+public class OkunCreatePipeline extends BaseTest {
+
+    private void deleteProjectPipeline() {
+
+        getDriver().findElement(By.xpath("//div[@id='tasks']/div/span[@class='task-link-wrapper ']" +
+                        "/a[@data-message='Delete the Pipeline ‘Bedryks’?']"))
+                .click();
+        getDriver().switchTo().alert().accept();
+
```
Comment (anonymoustuna) [inline src/test/java/OkunCreatePipeline.java:15]: Empty line.

File: src/test/java/OkunCreatePipeline.java [hunk 6]
```diff
@@ -0,0 +1,48 @@
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import runner.BaseTest;
+
```
Comment (anonymoustuna) [inline src/test/java/OkunCreatePipeline.java:5]: Empty line.

**Review by anonymoustuna — APPROVED** [review]

File: src/test/java/OkunCreatePipeline.java (hunk 1 above)
Comment (IrynaOkun) [reply src/test/java/OkunCreatePipeline.java:45]: Thank you for comments. I will correct it in the next commit

File: src/test/java/OkunCreatePipeline.java (hunk 5 above)
Comment (IrynaOkun) [reply src/test/java/OkunCreatePipeline.java:15]: Thank you for approve

**Review by jbioshock1912 — APPROVED** [review]

**Comment by IrynaOkun** [comment]
[AT_017.016| Creating Pipeline Project](https://trello.com/c/NAMKXqJ7/449-at017016-creating-pipeline-project)

**Comment by IrynaOkun** [comment]
[AT_017.016| Creating Pipeline Project](https://trello.com/c/NAMKXqJ7/449-at017016-creating-pipeline-project)

**Comment by IrynaOkun** [comment]
[AT_017.016| Creating Pipeline Project](https://trello.com/c/NAMKXqJ7/449-at017016-creating-pipeline-project)

**Pull request merged**